kingfisher_scrapy/spiders/honduras_portal_bulk.py

```diff
@@ -41,39 +45,48 @@ class HondurasPortalBulk(PeriodicSpider):
     # SimpleSpider
     data_type = 'release_package'
 
-    # PeriodicSpider
-    pattern = 'http://www.contratacionesabiertas.gob.hn/api/v1/descargas/{}'
-    formatter = staticmethod(components(-1))
+    available_publishers = {'oncae': 'Oficina Normativa de Contratación y Adquisiciones del Estado (ONCAE) / Honduras',
+                            'sefin': 'Secretaria de Finanzas de Honduras'}
+    available_systems = {'HC1': 'HonduCompras 1.0 - Módulo de Difusión de Compras y Contrataciones',
+                         'CE': 'Catálogo Electrónico',
+                         'DDC': 'Módulo de Difusión Directa de Contratos'}
 
-    available_publishers = {'oncae': 'oficina_normativa', 'sefin': 'secretaria_de_fin_HN.SIAFI2'}
-    available_systems = {'HC1': 'honducompras-1', 'CE': 'catalogo-electronico', 'DDC': 'difusion-directa-contrato'}
+    def start_requests(self):
+        url = 'http://www.contratacionesabiertas.gob.hn/api/v1/descargas/?format=json'
+        yield scrapy.Request(url, meta={'file_name': 'list.json'}, callback=self.parse_list)
 
     @classmethod
     def from_crawler(cls, crawler, publisher=None, system=None, *args, **kwargs):
         spider = super().from_crawler(crawler, publisher=publisher, system=system, *args, **kwargs)
-        if publisher and spider.publisher not in spider.available_publishers:
+        if publisher and spider.publisher not in spider.available_publishers.keys():
             raise SpiderArgumentError(f'spider argument `publisher`: {spider.publisher!r} not recognized')
 
         if system:
             if spider.publisher != 'oncae':
                 raise SpiderArgumentError(f'spider argument `system` is not supported for publisher: '
                                           f'{spider.publisher!r}')
-            if spider.system not in spider.available_systems:
+            if spider.system not in spider.available_systems.keys():
```

Comment:
```suggestion
            if spider.system not in spider.available_systems:
```

.keys() is the default behavior

Reply:
Yes, you're right!. I will remove it on the next commit